process/transaction/shardProcess.go

```diff
@@ -865,10 +862,8 @@ func (txProc *txProcessor) processRelayedTx(
 func (txProc *txProcessor) computeRelayedTxFees(tx, userTx *transaction.Transaction) relayedFees {
 	relayerFee := txProc.economicsFee.ComputeMoveBalanceFee(tx)
 	totalFee := txProc.economicsFee.ComputeTxFee(tx)
-	_, dstShardTxType := txProc.txTypeHandler.ComputeTransactionType(userTx)
-	isMoveBalance := dstShardTxType == process.MoveBalance
-	if txProc.enableEpochsHandler.IsFlagEnabled(common.FixRelayedMoveBalanceFlag) && isMoveBalance {
```

Comment:
there was a check for && isMoveBalance. Is it ok that it's not used any more? 

New lines 1163 and 1170 on executeFailedRelayedUserTx still check for isMoveBalance.

Reply:
Good catch. I think it needs to be deleted from there as well.

We should remove that isMoveBalance check from everywhere when we speak about relayed TX.  As you want to process the baseFee (gasPrice*(50K + len(data)*1500)) for every inner operation, as it happens on any tx.

Reply:
right, pushed

Reply:
ok!